set.mm

```diff
@@ -472151,6 +472186,240 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
+    sumsetsnorb.6 $e |- ( ph -> A C_ B ) $.
+    sumsetsnorb.7 $e |- ( ph -> X e. B ) $.
+    $( The sumset of a group with a single element is the element's orbit by
+       the group action.  See ~ gaorb .  (Contributed by Thierry Arnoux,
+       21-Jan-2024.) $)
+    sumsetsnorb $p |- ( ph -> ( .+^ " ( A X. { X } ) ) = [ X ] .~ ) $=
+      ( vh wcel vk csn cxp cima cec cv snssd mndsumsetss sselda wss crn imassrn
+      df-ec cpr co wceq wrex copab rneqi wex cab rnopab vex prss biimpri simprd
+      wa adantr exlimiv abssi eqsstri sstri a1i wbr w3a anim1i biantrurd df-3an
+      eqsstrid syl6bbr gaorb syl6rbbr cvv wb elecg sylancr elgrpsumsetsn rexbii
+      eqcom syl6bb 3bitr4rd eqrdav ) AUAGDKUBZUCUDZKHUEZEAWNEUAUFZAEGDWMJLNPQAK
+      ERUGUHUIAWOEWPAWOHWMUDZEKHUMWQEUJAWQHUKZEHWMULWRBUFZCUFZUNEUJZIUFWSFUOWTU
+      PIDUQZVGZBCURZUKZEHXDOUSXEXCBUTZCVAEXCBCVBXFCEXCWTETZBXAXGXBXAWSETZXGXHXG
+      VGXAWSWTEBVCCVCVDVEVFVHVIVJVKVKVLVMVSUIAWPETZVGZKWPHVNZSUFKFUOZWPUPZSDUQZ
+      WPWOTZWPWNTZXJXNKETZXIXNVOZXKXJXNXQXIVGZXNVGXRXJXSXNAXQXIRVPVQXQXIXNVRVTB
+      CKWPFHISDEOWAWBXJWPWCTXQXOXKWDUAVCAXQXIRVHZWPKHWCEWEWFXJXPWPXLUPZSDUQXNXJ
+      SDEFGJKWPLMNADEUJXIQVHXTWGYAXMSDWPXLWIWHWJWKWL $.
+  $}
+
+  ${
+    $d .X. x y $.  $d .x. x y $.  $d B x y $.  $d E x y $.  $d F x y $.
+    $d R x y $.  $d Z x y $.  $d ph x y $.
+    elringsumset.1 $e |- B = ( Base ` R ) $.
+    elringsumset.2 $e |- .x. = ( .r ` R ) $.
+    elringsumset.3 $e |- G = ( mulGrp ` R ) $.
+    elringsumset.4 $e |- .X. = ( +f ` G ) $.
+    elringsumset.5 $e |- ( ph -> E C_ B ) $.
+    elringsumset.6 $e |- ( ph -> F C_ B ) $.
+    $( Membership to a sumset for a ring operation.  (Contributed by Thierry
+       Arnoux, 20-Jan-2024.) $)
+    elringsumset $p |- ( ph -> ( Z e. ( .X. " ( E X. F ) ) <-> E. x e. E E. y
+       e. F Z = ( x .x. y ) ) ) $=
+      ( mgpbas mgpplusg elgrpsumset ) ABCDFGHIJKDEJNLREFJNMSOPQT $.
+  $}
+
+  ${
+    ringsumsetss.1 $e |- B = ( Base ` R ) $.
+    ringsumsetss.3 $e |- G = ( mulGrp ` R ) $.
+    ringsumsetss.4 $e |- .X. = ( +f ` G ) $.
+    ringsumsetss.5 $e |- ( ph -> R e. Ring ) $.
+    ringsumsetss.6 $e |- ( ph -> E C_ B ) $.
+    ringsumsetss.7 $e |- ( ph -> F C_ B ) $.
+    $( Closure for the sumset multiplicative operation in a ring.  (Contributed
+       by Thierry Arnoux, 20-Jan-2024.) $)
+    ringsumsetss $p |- ( ph -> ( .X. " ( E X. F ) ) C_ B ) $=
+      ( mgpbas crg wcel cmnd ringmgp syl mndsumsetss ) ABDEFGBCGIHNJACOPGQPKCGI
+      RSLMT $.
+  $}
+
+  ${
+    $d .X. x y $.  $d A x $.  $d B x y $.  $d K x y $.  $d R x y $.
+    $d X x y $.  $d h ph $.  $d h x $.  $d ph x y $.
+    sumsetsnpridl.1 $e |- B = ( Base ` R ) $.
+    sumsetsnpridl.2 $e |- G = ( mulGrp ` R ) $.
+    sumsetsnpridl.3 $e |- .X. = ( +f ` G ) $.
+    sumsetsnpridl.4 $e |- K = ( RSpan ` R ) $.
+    sumsetsnpridl.5 $e |- ( ph -> R e. Ring ) $.
+    sumsetsnpridl.6 $e |- ( ph -> X e. B ) $.
+    $( The sumset of a ring with a single element is the element's principal
```

Comment:
Here, in my opinion, the term "sumset" is even more confusing :-) Maybe something like "The product of the ring with a single element is equal to the principal ideal generated by that element".

Reply:
Done with last commit.